program-runtime/src/loaded_programs.rs

```diff
@@ -455,6 +455,20 @@ pub struct LoadedPrograms {
     /// Environments of the current epoch
     pub environments: ProgramRuntimeEnvironments,
     pub stats: Stats,
+    fork_graph: Option<Arc<RwLock<FG>>>,
```

Comment:
This should to be a [Weak](https://doc.rust-lang.org/std/sync/struct.Weak.html) reference.

Reply:
Funny that I came across the same problem now: https://github.com/anza-xyz/agave/pull/1893